### ISSUE TITLE
Is there a docker image or dockerfile that can be used to setup FAST-Pathology

### ISSUE THREAD

**Issue by lomshabhishek** [AICAN-Research/FAST-Pathology#90]
Facing issues while installing in Docker env
Docker image or file could come in handy

**Comment by smistad** [comment]
@andreped might have some docker images. 
You made some for FAST right?

**Comment by andreped** [comment]
> You made some for FAST right?

Yes, but for whatever reason, I get some issues with OpenCL when trying to access it.

Here is an example docker image that I have used together with Digital Slide Archive:
https://github.com/andreped/FP-DSA-plugin/blob/main/dsa/Dockerfile

At least it fails on all setups I have tried. I no longer have access to the Ubuntu-18.04 desktop which I tested this on the first time. Perhaps you see what is wrong, @smistad?
